settings.gradle.kts

```
@@ -36,6 +36,7 @@ include("detekt-rules-performance")
 include("detekt-rules-ruleauthors")
 include("detekt-rules-style")
 include("detekt-test")
+include("detekt-test-junit5")
```

```suggestion
include("detekt-test-junit")
```

Since this is the first release of this new module, JUnit 5 is the current release, and we're not using any deprecated APIs so should be safe to update to JUnit 6 when the time comes without having to make changes, I think we just call it `detekt-test-junit`. If we need a separate project for JUnit 6 then we can add the 6 suffix to that.

I did do a test build with JUnit 6 but ran into problems thanks to our build config but I can revisit that and see if it passes if we want to confirm before making a decision.

---

Renamed :)